docs/source/getting_started.rst

```diff
@@ -0,0 +1,296 @@
+
+Next steps
+------
+
+In this part, we will take a more advanced example to demonstrate how to use Results and Redis to gather results from a group and pipe them to a single actor.
+
+First, you will need to install the dependencies for using Remoulade with Redis::
+
+   $ pip install -U 'remoulade[redis]'
+
+You will also need to install Redis_ or use Docker_ to run a Redis image::
+
+   $ docker run -d --name redis -p 6379:6379 redis
+
+.. _Redis: https://redis.io/
+
+In this example, we want to get the weather for multiple cities, and then load them all into a file by using one single function instead of calling multiple times the ``load_city`` function. In order to achieve this, we have to write a new function : ``load_cities``, which takes a list of text to load instead of a single text::
+
+   @remoulade.actor
+   def load_cities(city_lines):
+       file = open("weather_db.txt", "a")
+       for text in text_list:
+           file.write(text)
+       file.close()
+
+Our get_weather file should now look like this :
+
+.. code-block:: python
+   :caption: get_weather.py
+   :emphasize-lines: 4, 5, 28, 30, 31, 33
+
+
+   import requests
+   import remoulade
+   from remoulade.brokers.rabbitmq import RabbitmqBroker
+   from remoulade.results.backends import RedisBackend
+   from remoulade.results import Results
+
+
+   @remoulade.actor
+   def extract_city(city):
+      ...
+
+
+   @remoulade.actor()
+   def transform_city(response):
+       ...
+
+
+   @remoulade.actor
+   def load_city(text):
+      ...
+
+
+   @remoulade.actor
+   def load_cities(city_lines):
+       ...
+
+
+   result_backend = RedisBackend()
+   rabbitmq_broker = RabbitmqBroker(url="/")
+   result_time_limit_ms = 10 * 60 * 1000 # 10 mn
+   rabbitmq_broker.add_middleware(Results(backend=result_backend, store_results=True, result_ttl=result_time_limit_ms))
+   remoulade.set_broker(rabbitmq_broker)
+   remoulade.declare_actors([extract_city, transform_city, load_city, load_cities])
+
+You will need to restart your Remoulade workers once again after adding this new function.
+Note that we set a time limit, and that in order to be able to gather the results from ``transform_city``, we have to specify "store_results=True" in the Results instantiation.
+A RedisBackend was also instantiated and added to the broker, and ``load_cities`` was declared to the broker.
+
+Our objective can now be achieved by running the following commands::
+
+   >>> from get_weather import extract_city, transform_city, load_cities
+   >>> from remoulade import pipeline, group
+   >>> cities = ['Paris', 'Lyon', 'Marseille', 'Lille', 'Bastia']
+   >>> grp = group([extract_city.message(city) | transform_city.message() for city in cities])
+   >>> weather_etl_pipeline = grp | load_cities.message()
+   >>> weather_etl_pipeline.run()
+
+The worker terminal now shows the pipeline has been successfully run and the correct data has been written into the weather file::
+
+   Paris:
```

Comment:
remove or at least put the appropriate line break (and cities). But I think we can remove this part